pkg/flare/config_check.go

```diff
@@ -95,10 +95,10 @@ func PrintConfig(w io.Writer, c integration.Config) {
 	}
 	for _, inst := range c.Instances {
 		var ID string
-		if len(c.Instances) == 1 {
```

Comment:
at this point we should go directly with `ID := string(check.BuildID(c.Name, inst, c.InitConfig))` for everyone

Reply:
Yes, I think it's better